File: 251108/특정 수 맞히기/catching-specific-number.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim().split('\n');
const nums = input.map(Number);
let i = 0
while (true) {
    if (nums[i] < 25) console.log('Higher');
    else if (nums[i] > 25) console.log('Lower');
    else {
        console.log('Good');
        break;
    }
    i += 1;
}

File: 260312/특정 수 맞히기/catching-specific-number.py
import sys

input = sys.stdin.readline

# 선언부
def program(n: int):
    if n < 25:
        return 'Higher'
    elif n > 25:
        return 'Lower'
    else:
        return 'Good'

# 입력부
while True:
    i = int(input())
    result = program(i)
    print(result)
    if result == 'Good':
        break

# 호출부
